tests/darcy_velocity_postprocessor.prm

```diff
@@ -0,0 +1,6 @@
+include $ASPECT_SOURCE_DIR/tests/darcy_convection_step.prm
```

Comment:
In this case, I would say it's probably better to just add the lines below to the darcy_convection_step.prm file (rather than adding this as a new test), in particular because that file has a description on what the darcy velocity should be, so the values can be checked easily.